2018q4/powerparity.md

```diff
@@ -0,0 +1,28 @@
+## FreeBSD on Power9 (ppc64) Parity ##
+
+   - NMI semantics: NMIs need to be emulated by only soft disabling interrupts,
```

Comment:
There's no errors in this that I can spot, but it's unclear to me whether the issue of NMIs needing to be emulated… et cetera has been fixed - perhaps you can rephrase it ever so slightly to indicate whether this is something that's still being worked on, or whether it's solved?